From 3c9f42940c96f2bd919aa4b9d5d82336781bbdd7 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:39:20 -0400
Subject: [ethernet] i40e: fix a stray print message

Message-id: <1409658042-22689-200-git-send-email-sassmann@redhat.com>
Patchwork-id: 90399
O-Subject: [RHEL7.1 PATCH 199/281] i40e: fix a stray print message
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Shannon Nelson <shannon.nelson@intel.com>

This log print message will probably never be seen, but it needs to match
the "attempting to rebuild switch\n" log message a few lines above.

Change-ID: Ic3f5b4f67568d721cb02e826cf2cb33847f51c11
Signed-off-by: Shannon Nelson <shannon.nelson@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit cde4cbc7800469c9b8424261b69c45a71a89caf4)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_main.c b/drivers/net/ethernet/intel/i40e/i40e_main.c
index 2a91e67..4cbdd7a 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_main.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_main.c
@@ -5775,7 +5775,7 @@ static void i40e_reset_and_rebuild(struct i40e_pf *pf, bool reinit)
  }
 
  if (pf->vsi[pf->lan_vsi]->uplink_seid == pf->mac_seid) {
-  dev_info(&pf->pdev->dev, "attempting to rebuild PF VSI\n");
+  dev_dbg(&pf->pdev->dev, "attempting to rebuild PF VSI\n");
   /* no VEB, so rebuild only the Main VSI */
   ret = i40e_add_vsi(pf->vsi[pf->lan_vsi]);
   if (ret) {
-- 
1.7.1